docs/experimental/ticket-auction.md

```diff
@@ -0,0 +1,258 @@
+---
+id: ticket-auction
+title: Building a Dutch Auction with Ticket based NFTs in Ligo
+sidebar_label: Ticket NFT Auction in Ligo
+---
+
+## Introduction
+Tickets are a feature added to Tezos in the Edo protocol upgrade that allow a smart contract to authenticate data with respect to a Tezos address. They provide a convenient mechanism  to grant portable permissions or issue tokens without requiring the token holder or permission grantee to interact with a centralized contract.
+
+Tickets' decentralized nature make them perfectly suited for representing assets for which users are in full control of their tokens. In this tutorial, we will create a Dutch/descending-price auction of ticket-based NFTs implemented in Ligo, along with ticket wallets for auction participants.
+## Set-up
+- Follow instructions in [Client Setup](/docs/setup/1-tezos-client) to set up
+  `tezos-client.` Also, make sure to point to a public Edo test network node such as `https://edonet-tezos.giganode.io` or to an Edo [sandbox](/docs/setup/2-sandbox#try-the-edo-protocol) and make sure to have at least 2 test accounts for Alice and Bob with funds.
+- Follow instructions on [Ligo](https://ligolang.org/docs/intro/installation) to install the most recent Ligo version.
+- Clone tutorial repo
+```shell
+$ git clone https://github.com/tqtezos/ticket-tutorials.git
+$ cd ./ticket-tutorials/tutorials/auction/ligo
+```
+- Compile Ligo code
+```shell
+$ ligo compile-contract --protocol edo --disable-michelson-typechecking nft_wallet.mligo main > ./michelson/nft_wallet.tz
+$ ligo compile-contract --protocol edo --disable-michelson-typechecking nft_auction.mligo main > ./michelson/nft_auction.tz
+```
+## NFT Wallet Contract
+The first contract used in this demo is an NFT wallet that will create, destroy, send, and receive NFTs. For this demo, it will also be used to configure NFT Auctions that will be performed in our NFT Auction contract.
+
+Every NFT created with the wallet also includes some token metadata, following the standard specified for FA2 in [TZIP-12](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-12/tzip-12.md#token-metadata-values) in which token metadata is stored as a Michelson value of type `(map string bytes)`.
+
+### Originations
+
+#### Alice
+
+Alice originates her nft-wallet contract and initializes herself as the admin, the wallet to have no tickets, the ticket_id that will be incremented after every NFT is minted to be 0, and no ticket metadata yet as the wallet does not contain tickets.
+
+```sh
+$ tezos-client originate contract nft-wallet transferring 0 from alice \
+       running "$(pwd)/michelson/nft_wallet.tz" \
+        --init "Pair \"tz1LrQB9HrJcUaD9NKEvV65tnaiU8trPXwmt\" (Pair {} (Pair 0 {}))" \
+        --burn-cap 0.305
+
```

Comment:
I think it might be useful to make it more similar in style to `FA1.2 Lorentz` tutorial: use env variables for alice and bob tz addresses and provide detailed explanation of init storage